### DISCUSSION TITLE
How to get data from edgeX service by external python application

### DISCUSSION THREAD

**Discussion by anhtuan85** [edgexfoundry/community#313] [Q&A]
Hi everyone.
I have installed EdgeX and data is sent to device services. How can I access this data via Python from another address (not localhost)?
I just learned EdgeX recently.

**Comment by jiekechoo** [comment]
There is no python sdk in EdgeX currently.
You can use `Export` features in the App Service. [MQTTExport](https://docs.edgexfoundry.org/3.1/microservices/application/services/AppServiceConfigurable/details/AvailablePipelineFunctions/#mqttexport) and [HTTPExport](https://docs.edgexfoundry.org/3.1/microservices/application/services/AppServiceConfigurable/details/AvailablePipelineFunctions/#httpexport)
